Makefile

```diff
@@ -56,7 +56,7 @@ else
 
     # Resolve all symbols at link time
     ifeq ($(SYS),Linux)
-    	CFLAGS += -DX11_GAMMA_WORKAROUND
+        CFLAGS += -DX11_GAMMA_WORKAROUND -fno-tree-vectorize
```

Comment:
tree vectorization doesn't get enabled until O3 anyway so unless someone explicitly sets O3 on it shouldn't change anything.  I do get a small (somewhere around 5%) speed increase at O3, and this is the only optimization enabled by it that breaks anything.